src/config.js

```diff
@@ -9,7 +9,7 @@ export default {
     endpoints: {
       status: '/status'
     },
-    baseUrl: __INJECTED_CONFIG.dashboard.apiBaseUrl || 'http://localhost:8000/api'
+    baseUrl: __INJECTED_CONFIG.dashboard.apiBaseUrl || 'http://localhost:8000/api/status'
```

Comment:
This was supposed to be the `baseUrl` for the API. You're adding the endpoint `/status` in it too. It should just be `'http://localhost:8000/api'`.